web-app/resources/views/app.blade.php

```diff
@@ -22,8 +22,8 @@
                 <img src="/logo-banner.png" height="120px">
             </div>
             <div class="col text-center text-light header-title">
-                <h3 class="d-none d-md-block" >Electronic Contract Administration System (ECAS)</h3>
-				<h3 class="d-block d-md-none" >ECAS</h3>
+                <h3 class="header-title-desktop" >Electronic Contract Administration System (ECAS)</h3>
+				<h3 class="eader-title-mobile" >ECAS</h3>
```

Comment:
typo I think you meant header?

Reply:
Sorry typo error, surprisingly it didn't break the fix but I fixed anyway and pushed my latest changes

Reply:
benefits of peer review heh.